index.d.ts

```diff
@@ -54,6 +54,8 @@ export interface HandleResponsesOptions {
 	alwaysServeDocs?: boolean
 
 	specOutputFileBehavior: SPEC_OUTPUT_FILE_BEHAVIOR | string
+
+  swaggerDocumentOptions: Array<string>
```

Comment:
I did some looking up and found this package that has documented the types for `swagger-ui-express` (as often is with more popular packages) - [1]

you can apply the following patch (copy, save to file e.g. `a.patch`)

```sh
# write patch to file a.patch, then:
git apply a.patch
```

```patch
diff --git a/index.d.ts b/index.d.ts
index d031f56..769b2df 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -9,11 +9,13 @@
 
 import { Express } from 'express';
 import { OpenAPIV2, OpenAPIV3 } from 'openapi-types';
+import { SwaggerUiOptions } from "swagger-ui-express"
 
 /** re-export for ease of use for the end user */
 export {
 	OpenAPIV2,
-	OpenAPIV3
+	OpenAPIV3,
+	SwaggerUiOptions
 };
 
 export enum SPEC_OUTPUT_FILE_BEHAVIOR {
@@ -48,14 +50,14 @@ export interface HandleResponsesOptions {
 	tags?: Array<string>;
 
 	/** Ignored node environments */
-	ignoredNodeEnvironments?: Array<string>
+	ignoredNodeEnvironments?: Array<string>;
 
 	/** Always serve api docs */
-	alwaysServeDocs?: boolean
+	alwaysServeDocs?: boolean;
 
-	specOutputFileBehavior: SPEC_OUTPUT_FILE_BEHAVIOR | string
+	specOutputFileBehavior: SPEC_OUTPUT_FILE_BEHAVIOR | string;
 
-  swaggerDocumentOptions: Array<string>
+  swaggerDocumentOptions: SwaggerUiOptions;
 }
 
 /**
diff --git a/index.js b/index.js
index dd73dbd..dd9b1c5 100644
--- a/index.js
+++ b/index.js
@@ -62,6 +62,9 @@ let ignoredNodeEnvironments;
 let serveDocs;
 let specOutputPath;
 let specOutputFileBehavior;
+/**
+ * @type { typeof import('./index').SwaggerUiOptions }
+*/
 let swaggerDocumentOptions;
 
 /**
@@ -369,7 +372,7 @@ function handleResponses(expressApp,
     ignoredNodeEnvironments: DEFAULT_IGNORE_NODE_ENVIRONMENTS,
     alwaysServeDocs: undefined,
     specOutputFileBehavior: SPEC_OUTPUT_FILE_BEHAVIOR.RECREATE,
-    swaggerDocumentOptions: []
+    swaggerDocumentOptions: {}
   }) {
 
   ignoredNodeEnvironments = options.ignoredNodeEnvironments || DEFAULT_IGNORE_NODE_ENVIRONMENTS;
@@ -490,7 +493,7 @@ function handleRequests() {
 /**
  * @type { typeof import('./index').init }
  */
-function init(aApp, aPredefinedSpec = {}, aSpecOutputPath = undefined, aWriteInterval = 0, aSwaggerUiServePath = DEFAULT_SWAGGER_UI_SERVE_PATH, aMongooseModels = [], aTags = undefined, aIgnoredNodeEnvironments = DEFAULT_IGNORE_NODE_ENVIRONMENTS, aAlwaysServeDocs = undefined, aSpecOutputFileBehavior = SPEC_OUTPUT_FILE_BEHAVIOR.RECREATE, aSwaggerDocumentOptions = []) {
+function init(aApp, aPredefinedSpec = {}, aSpecOutputPath = undefined, aWriteInterval = 0, aSwaggerUiServePath = DEFAULT_SWAGGER_UI_SERVE_PATH, aMongooseModels = [], aTags = undefined, aIgnoredNodeEnvironments = DEFAULT_IGNORE_NODE_ENVIRONMENTS, aAlwaysServeDocs = undefined, aSpecOutputFileBehavior = SPEC_OUTPUT_FILE_BEHAVIOR.RECREATE, aSwaggerDocumentOptions = {}) {
   handleResponses(aApp, {
     swaggerUiServePath: aSwaggerUiServePath,
     specOutputPath: aSpecOutputPath,
diff --git a/package.json b/package.json
index a0eff48..a2af942 100644
--- a/package.json
+++ b/package.json
@@ -45,6 +45,7 @@
     "winston": "^3.3.3"
   },
   "devDependencies": {
+    "@types/swagger-ui-express": "^4.1.2",
     "body-parser": "^1.18.2",
     "bson": "^4.0.4",
     "coveralls": "^3.0.0",
```

you can then run `npm install` to get the dependencies up if needed.

[1] https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/swagger-ui-express

Reply:
very good @kiprasmel 
I applied the changes you proposed.